frame/pablo/src/lib.rs

```diff
@@ -284,10 +290,19 @@ pub mod pallet {
 
 		/// Required origin for pool creation.
 		type PoolCreationOrigin: EnsureOrigin<Self::Origin>;
+
+		/// Time provider.
+		type Time: Time;
+
+		/// The interval between TWAP computations.
+		#[pallet::constant]
+		type TWAPInterval: Get<MomentOf<Self>>;
 	}
 
 	#[pallet::pallet]
 	#[pallet::generate_store(pub(super) trait Store)]
+	// TODO(hussein-aitlahcen): MaxEncodedLen not yet implemented for trait AT `Time::Moment`.
+	#[pallet::without_storage_info]
```

Comment:
What would be the effect of this?

Reply:
Can we just use `#[pallet::unbounded]` on the individual storage item that is affected here like it suggests at https://docs.substrate.io/v3/runtime/macros/#palletgenerate_storage_info